examples/app_with_observability/app_nested_async.py

```diff
@@ -107,7 +107,7 @@ async def wrapper(context_1: str, max_tokens: int):
     return final_response
 
 
-@ag.entrypoint()
+@ag.entrypoint
```

Comment:
```suggestion
@ag.entrypoint()
```

Tracing won't work if you don't call the decorator class.

Reply:
see previous comment